server/routes/notification.js

```diff
@@ -220,6 +221,24 @@ router.post("/price", async (req, res) => {
 
     await transaction.commit();
 
+    //Hashmap for keeping track who are the users that we already sent an event to
+    let notifiedUsers = {};
+
+    //We iterate the userLists which contains all the affected users of the price change.
+    //We check here if the user is NOT in the notifiedUsers hashmap, if so we emit an event
+    //and add the userId to the hashmap so that if the users userId comes up again, we dont 
+    //send them another emit again.
+    userLists.forEach((userList) => {
+      let stringUserId = `${userList.userId}`;
+
+      if (userSockets.connections[stringUserId]) {
+        if (!notifiedUsers[stringUserId]) {
```

Comment:
could use one if statement and combine the two conditions with an `&&` operator. This would be cleaner imo.